src/cms/templates/push_notifications/push_notification_form.html

```diff
@@ -22,76 +38,71 @@ <h2 class="heading font-normal">
-                {% render_field push_notification_form.channel placeholder=channel_placeholder class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
-            </div>
-            <div class="w-full p-4 mb-4 rounded border border-solid border-gray-200 shadow bg-white">
-                <label for="id_title" class="block mb-2 font-bold cursor-pointer">{% trans 'Title' %}</label>
-                {% blocktrans asvar title_placeholder %}Insert title in {{ language }} here{% endblocktrans %}
-                {% render_field push_notification_translation_form.title placeholder=title_placeholder class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
-            </div>
-            <div class="w-full p-4 mb-4 rounded border border-solid border-gray-200 shadow bg-white">
-                <label for="id_text" class="block mb-2 font-bold cursor-pointer">{% trans 'Content' %}</label>
-                {% blocktrans asvar text_placeholder %}Insert content in {{ language }} here{% endblocktrans %}
-                {% render_field push_notification_translation_form.text placeholder=text_placeholder class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
-            </div>
-        </div>
-        <div class="w-1/3 pl-2">
-            <div class="w-full px-4 py-2 rounded border border-solid border-gray-200 shadow bg-white">
-                <div class="py-2 border-b solid border-gray-200 mb-2">
-                    <span class="block mb-2 font-bold">{% trans 'Language' %}</span>
-                    <label class="text-xs uppercase block">{% trans 'Current language' %}</label>
-                    <div class="relative my-2">
-                        <img src="{% static '' %}images/flags/{{ language.code }}.png" title="{{ language.name }}" />
-                        {{ language.name }}
+        <div class="w-2/3 flex flex-wrap flex-col pr-2 tabbed">
+            {{page_form.errors}}
+            {{page_translation_form.errors}}
+            <ul class="flex flex-wrap pl-4">
+                {% for other_language in languages %}
+                <li id="li-{{ other_language.id }}" class="mr-1 {% if other_language.id == language.id %}z-10{% endif %}" style="margin-bottom: -2px" onclick="switch_language('{{ other_language.id }}')">
+                        <div class="bg-white text-blue-500 hover:bg-blue-500 hover:text-white border-l-2 border-t-2 border-r-2 border-blue-500 font-bold rounded-t-lg">
+                            <div class="border-b-2 border-white">
+                                <div class="p-4">{{ other_language }}</div>
```

Comment:
```suggestion
                                <div class="p-4">{{ other_language.translated_name }}</div>
```
I propose to use `other_language.translated_name` here to display the name of the language in the current backend language.

Reply:
Done